resources/metadata/jsBuild.yaml

```diff
@@ -0,0 +1,42 @@
+metadata:
+  name: nodeJsBuild
```

Comment:
shouldn't we use a similar pattern like for `maven` and `mta`?

e.g. `npmExecute`, `npmBuild`, ...

Reply:
My thinking was that it is not really limited to npm, we also support yarn lockfiles, and we might need to support more nodejs package managers. The name expresses that it builds node applications with whatever the right tool is.

But I'm not dogmatic on that, so if we come to the conclusion that it should be `npmBuild`, then we still can change that before the merge.

Reply:
Renamed